redirects.csv

```diff
@@ -135,3 +135,4 @@ https://discipleship.crossroads.net/*,https://www.crossroads.net/huddle,301!,mas
 /healing/divorcecare,/groups/onsite-healing-groups/divorce-care,301!
 /healing/postpartum-healing,/groups/onsite-healing-groups/postpartum,301!
 /*,/404.html,404
+/groups/onsite-groups/*,/groups/onsite,301!
```

Comment:
@dholloran-ample This rule won't ever be evaluated b/c it falls after the wildcard rule. Can you move it up by one line?